etp-backend/src/main/sql/solita/etp/db/viesti.sql

```diff
@@ -25,6 +25,22 @@ where from_id = :laatija-id or vastaanottajaryhma_id = 1 or
 order by viestiketju.id desc
 limit :limit offset :offset;
 
+-- name: select-viestiketjut-for-laskuttaja
```

Comment:
helpompi muokata: select-viestiketjut-for-laatija -> select-viestiketjut-for-group ja lisätä bind-parametri ryhmälle

Reply:
eli bind-parametrit olisivat: :kayttaja-id + :vastaanottajaryhma-id

Reply:
sitten funktio: rooli-id->vastaanottajaryhma-id

Reply:
Hyvä idea. Pistin tähän päälle commitin joka yhdistää nuo funkkarit.